### PR TITLE
Consider cells with whitespaces only as blank cells

### PR DESCRIPTION
**Pull request by mmenestret**
## Summary
When parsing Excel cells, consider cells with only whitespaces as blank to avoid human mistakes.

### CODE REVIEW

File: src/main/scala/com/ebiznext/comet/schema/generator/XlsReader.scala [hunk 1]
```diff
@@ -18,21 +17,27 @@ class XlsReader(path: String) {
...
     workbook.getSheet("domain").asScala.drop(1).headOption.flatMap { row =>
       val nameOpt = Option(row.getCell(0, Row.MissingCellPolicy.RETURN_BLANK_AS_NULL))
-        .map(formatter.formatCellValue)
+        .flatMap(formatter.formatCellValue)
       val directoryOpt = Option(row.getCell(1, Row.MissingCellPolicy.RETURN_BLANK_AS_NULL))
-        .map(formatter.formatCellValue)
+        .flatMap(formatter.formatCellValue)
       val ack = Option(row.getCell(2, Row.MissingCellPolicy.CREATE_NULL_AS_BLANK))
-        .map(formatter.formatCellValue)
+        .flatMap(formatter.formatCellValue)
```
Comment (mhdkassir) [inline src/main/scala/com/ebiznext/comet/schema/generator/XlsReader.scala:38]: Please make sure that `ack `is still set to "" in the generated yaml when the cell is empty.

Comment (mmenestret) [reply src/main/scala/com/ebiznext/comet/schema/generator/XlsReader.scala:38]: I'll double check that behavior

Comment (mmenestret) [reply src/main/scala/com/ebiznext/comet/schema/generator/XlsReader.scala:38]: Don't you think that:

```def getAck(): String = ack.map(ack => if (ack.nonEmpty) "." + ack else ack).getOrElse(".ack")```

Is enough ?

**Comment by hayssams** [comment]
Why is this PR still waiting to be merged ?

**Pull request merged**